e2e/tests/judiciaryAdministersCaseAfterSubmission_test.js

```diff
@@ -33,6 +33,7 @@ Scenario('Judiciary adds allocated judge', async (I, caseViewPage, allocatedJudg
   caseViewPage.selectTab(caseViewPage.tabs.casePeople);
   I.seeInTab(['Allocated Judge', 'Judge or magistrate\'s title'], 'Her Honour Judge');
   I.seeInTab(['Allocated Judge', 'Last name'], 'Moley');
+  I.seeInTab(['Allocated Judge', 'Email Address'], 'moley@example.com');
```

Comment:
I think I expect to see somewhere in e2e that user cannot see judge email address

Reply:
Added test for LA - not seeing judge email address.

Reply:
> I think I expect to see somewhere in e2e that user cannot see judge email address

Updated tests with LA user who cannot see the judge email